packages/mds-web-sockets/server.ts

```diff
@@ -65,14 +61,15 @@ export const WebSocketServer = () => {
         if (clients.isAuthenticated(ws)) {
           if (args.length === 2) {
             const [entity, payload] = args
+            // Limit messages to only supported entities
             switch (entity) {
```

Comment:
Note that entity type is still checked and only blessed ones are passed on to clients